kola/tests/misc/falco.go

```diff
@@ -0,0 +1,27 @@
+package misc
+
+import (
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+)
+
+// create a test case
+func init() {
+	register.Register(&register.Test{
+		Run:         loadFalco,
+		ClusterSize: 1,
+		Name:        "cl.misc.falco",
+		Distros:     []string{"cl"},
+		Platforms:   []string{"qemu"},
+		// selinux blocks insmod from within container
+		Flags: []register.Flag{register.NoEnableSelinux},
+	})
+}
+
+func loadFalco(c cluster.TestCluster) {
+	// load the falco binary
+	// TODO: first supported version will be 0.33.0, but use master tag for now
```

Comment:
On Flatcar side, does it work with all kernel version ? (5.15 and 5.10 for LTS)

Reply:
It's expected to work with all Flatcar versions, going as far back as when we started shipping a kernel module build env in /lib/modules/*/build.